docs/docker.md

```diff
@@ -0,0 +1,37 @@
+# Botbowl :heart: docker
+[Docker](http://docker.com) is a platform designed to help developers build, share, and run modern applications. 
+
+## Submitting a docker image to the Bot Bowl competition
+Your image will be started with `docker run -p <host_port>:5100 -t <bot_image>`. Below follows detailed instructions to build and submit your bot in a docker image. In the example, we'll build a bot called "nuffle". 
+
+
+Modify (../examples/containerized_bot.py) so it starts your bot instead of the scripted bot. 
+
+Add your additional dependencies into (../examples/extra_requirements.txt), if you're bot is based on the [A2C example](a2c.md), you want to add `torch` for example. 
+
+Build the docker image by calling this in the command line in **the root folder of this repository**: 
+```shell
+docker build . -t nuffle_bot_image --file docker/Dockerfile.comptetition_bot
```

Comment:
Is nuffle_bot_image a tag/name you give the image? Maybe add this, so people will know how to name it something else. Maybe their bot is not called nuffle.

Reply:
I added the clarification that it's a tag. Since [a tag is not a name in docker](https://stackoverflow.com/a/41521128/21993148), I won't say name here.